### PR TITLE
Fixed the name of the protocol variable

### PR DESCRIPTION
**Pull request by HernanRivasAcosta**
Fixed the name of the ``protocol`` variable in the example.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[8986.1](https://travis-ci.org/esl/MongooseIM/jobs/753351547) / Erlang 23.0.3 / small_tests / 3fa54d8afe77c03cdb3ec3651d737714ae393306
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/3000/8986/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/3000/8986/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-01-07_15.28.59/lib.mongooseim.logs/run.2021-01-07_15.28.59/suite.log.html)

File: doc/authentication-methods/anonymous.md [hunk 1]
```diff
@@ -30,5 +30,5 @@ Specifies the SASL mechanisms supported by the `anonymous` authentication method
```
Comment (leszke) [inline doc/authentication-methods/anonymous.md:30]: In line 17 up there, could you also update the example to `protocol = ...`?

**Review by leszke — COMMENTED** [review]
Hi Hernan,
Thanks for catching this. There's still one small thing though to be done here :)

File: doc/authentication-methods/anonymous.md (hunk 1 above)
Comment (HernanRivasAcosta) [reply doc/authentication-methods/anonymous.md:30]: Excellent catch, missed that one.

**Review by leszke — APPROVED** [review]
Thanks!

**Comment by mongoose-im** [comment]
[9005.1](https://travis-ci.org/esl/MongooseIM/jobs/753521302) / Erlang 23.0.3 / small_tests / 2eba21c29b2b47d97943320f3d04a7d6693be38e
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/3000/9005/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/3000/9005/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-01-08_12.52.58/lib.mongooseim.logs/run.2021-01-08_12.52.58/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks great!

**Pull request merged**